src/html/js/user.js

```diff
@@ -375,6 +375,87 @@ function getJoinInfoList(token) {
     });
 }
 
+function formatDate(date) {
+	var yyyymmdd =  date.getFullYear() + '/' +
+	          ('0' + (date.getMonth() + 1)).slice(-2) + '/' +
+	          ('0' + (date.getDate())).slice(-2);
+	return yyyymmdd;
+}
+
+function viewJoinConsiderList(entryFlag,articleId){
+    callArticleFunction(function (token,arg) {
+	    var oData = 'test_reply';
```

Comment:
Please use proper name even for prototype. You need to design the proper data structures (OData) first.